src/main/java/org/commonwl/view/workflow/Workflow.java

```diff
@@ -353,6 +355,16 @@ public void setLicenseLink(String licenseLink) {
     this.licenseLink = licenseLink;
   }
 
+  public String getLicenseName() {
+    if (licenseLink == null) {
+      return null;
+    }
+    if (licenseLink.startsWith(LicenseUtils.SPDX_LICENSES_PREFIX)) {
+      return SpdxLicense.fromId(licenseLink.replace(LicenseUtils.SPDX_LICENSES_PREFIX, "")).name;
+    }
+    return licenseLink;
```

Comment:
These lines lack test coverage

Reply:
Test added

Reply:
One more needed here, the non -SPDX case 😅